Applications/DataExplorer/NetCdfDialog/NetCdfConfigureDialog.cpp

```diff
@@ -21,16 +21,21 @@
 
 #include <vtkImageImport.h>
```

Comment:
Maybe a stupid question, but is this include still necessary?

Reply:
Yes, this is needed when we convert the netCDF data into image data.